fedimint-client/src/sm/executor.rs

```diff
@@ -836,6 +888,15 @@ impl ExecutorInner {
             .collect::<Vec<_>>()
             .await
     }
+
+    pub async fn log_event_dbtx<E, Cap>(&self, dbtx: &mut DatabaseTransaction<'_, Cap>, event: E)
+    where
+        E: Event + Send,
+        Cap: Send,
+    {
+        dbtx.log_event(self.log_ordering_wakeup_tx.clone(), None, event)
```

Comment:
nit: Having to provide the `wakeup_tx` manually is super weird DX, what does it even do? Why can't that sender be part of the dbtx? Is it related to the `log_event` fn being implemented through some sort of extension trait?